packages/builder/sample-project/scripts/0-sampleScript.js

```diff
@@ -1 +1,6 @@
-console.log("Hello world!")
+
+function run(runtimeEnv, deployer, accounts) {
+  console.log("Sample script has been executed!")
+}
```

Comment:
if this is an `algob run` script, than deployer is not provided (this has been written in the spec more than a week ago)

Reply:
Deployer is last param and can be `null`